upi/openstack/update_machineset.py

```diff
@@ -0,0 +1,19 @@
+import yaml;
+path = "openshift/99_openshift-cluster-api_worker-machineset-0.yaml"
+data = yaml.safe_load(open(path));
+networks = data['spec']['template']['spec']['providerSpec']['value'].get('networks', [])
+networks.append(
+    {
+        'filter': {},
+        'subnets': [
+            {
+                'filter': {
+                    'name': str(data['metadata']['labels']['machine.openshift.io/cluster-api-cluster'] + '-acicontainers-nodes'),
```

Comment:
Thanks for fixing the name here, this will supersede the change in https://github.com/noironetworks/openshift_installer/pull/100